### ISSUE TITLE
Change the editor default UI size value to match standards

### ISSUE THREAD

**Issue by fabriceci** [godotengine/godot-proposals#2980]
### Describe the project you are working on

This is a **tiny change** that I think it can improve the **default** feeling.

I always felt the interface a bit “zoomed”, I get rid of that by setting the UI font/code font from 14 to 12. I think it's a better default and increase the readability, especially in the node tree and the editor viewport.

Here is a comparison with well know software: before (check the title bar font size compared to the rest):

![before](https://user-images.githubusercontent.com/6397893/125195096-73a11980-e254-11eb-8323-6c6a5e1dde53.jpg)

Here is after:

![after](https://user-images.githubusercontent.com/6397893/125195053-3b99d680-e254-11eb-841e-2a74cd1c41b4.jpg)

(Sorry for not respecting the template, but it did not work for this type of proposal)

### Describe the problem or limitation you are having in your project

/

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

/

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

/

### If this enhancement will not be used often, can it be worked around with a few lines of script?

/

### Is there a reason why this should be core and not an add-on in the asset library?

/

**Comment by AFE-GmdG** [comment]
I don't think 12 is the better default value, because I find it harder to read - and i'm not that old :-)
This is a personal setting that you only set once.

**Comment by fabriceci** [comment]
> This is a personal setting that you only set once.

Indeed, but as you can see in the screenshot where I put next to Godot: Photoshop, Blender and VSCode, you can clearly see that 14 is much bigger than the default of these applications, that's the point. (is the same for you too I suppose ?)

**Comment by Calinou** [comment]
> I always felt the interface a bit “zoomed”, I get rid of that by setting the UI font from 14 to 12. I think it's a better default and increase the readability, especially in the node tree and the editor viewport.

I did this in https://github.com/godotengine/godot/pull/32061, but that PR was ultimately rejected. I doubt the status quo would change.

**Comment by fabriceci** [comment]
@Calinou not saw that PR, same idea :) (Note: the screenshot I made was on a laptop).

Yes, I understand that tastes are subjective are people don't like changes :). However, objectively, compared to other "standard" software (this applies too with Unity or UE), the Godot UI looks "zoomed" (like the scaled mode is on)

I think it's better to match standard, and then, everyone can tweak with his tastes.

(anyway not a big deals, one way or the other, this can be easily by changed in the settings)

**Comment by QbieShay** [comment]
I think i prefer to keep it bigger. Rationale being that bigger text is more accessibility friendly for people that have impaired vision.
Tiny text may make you do extra effort without noticing until you get headache.

For this reason, I think we should err on the side of big.

**Comment by YuriSizov** [comment]
At 14 current 3.5 RC7 build seems to be the same as Blender 3.2.1. I have a 3440x1440 screen with 125% system scaling, though.

![blender_2022-08-02_18-01-14](https://user-images.githubusercontent.com/11782833/182407530-f548c842-60d2-4a4f-a04e-13803bc2a799.png)

Any tinier, and I won't be able to read it with a headache.

**Comment by fabriceci** [comment]
@YuriSizov Actually your screenshot goes in my direction, as Godot doesn't take scaling into account, the correct size seems to be 25% smaller (+ need to take scaling into account).

**Comment by Calinou** [comment]
Taking the OS scale factor into account requires https://github.com/godotengine/godot-proposals/issues/2661 to be implemented first.

**Comment by Calinou** [comment]
@fabriceci Did you try the **Compact** spacing preset that's available in Godot 4.3? It's in **Interface > Theme > Spacing Preset** in the Editor Settings.
